docs/rationale.md

```diff
@@ -1,3 +1,8 @@
+---
+layout: default
+nav_order: 6
```

Comment:
I feel like this should be higher up in the list, but not sure where. Maybe just after the main page? Also, let's rename the heading too to also just be "Rationale"?